pkg/api/v1alpha1/trace_types.go

```diff
@@ -32,8 +32,8 @@ type ContainerFilter struct {
 	// Podname selects events from pods with these labels
 	Labels map[string]string `json:"labels,omitempty"`
 
-	// ContainerName selects events from containers with this name
-	ContainerName string `json:"containerName,omitempty"`
+	// Name selects events from containers with this name
+	Name string `json:"containerName,omitempty"`
```

Comment:
I'd avoid to change this one. `ContainerFilter` in this case refers to ways to filtering multiple containers, for instance all containers in a given kubernetes namespace, all containers with some given labels etc.  It's used like this:

```
  filter:
    namespace: default
    containerName: foo
```

If we rename this, then `name:` in that context won't be that clear. 

Btw, it's also the reason why the CI is failing -> https://github.com/kinvolk/inspektor-gadget/runs/3319475101?check_suite_focus=true

Reply:
My idea of changing this was because I don't like how looks comparing `ContainerFilter.ContainerName` with `ContainerSelector.Name`. I would prefer both use the same notation. However, I understand your point.